opendoors/config.py

```diff
@@ -3,6 +3,7 @@
 """
 import configparser
 import os
+import shutil
```

Comment:
Is anything from shutil used in this file?

Reply:
fixed